Contributing.md

```diff
@@ -0,0 +1,92 @@
+# Contribution Guide
+
+If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🥳
+
+# How to Contribute
+
+1. Fork this repository.
+
+2. Clone the forked repository.
+
+   ```bash
+   git clone 'https://github.com/bsoc-bitbyte/busify.git'
+   ```
+
+3. Navigate to the project directory.
+
+   ```bash
+   cd busify
+   ```
+
+4. Make changes in source code.
+
+5. Stage your changes and commit
+
+6. git add .
+
```

Comment:
Merge the 6th point with the 5th point because `git add .` will stage your changes.